coalib/results/result_actions/ShowPatchAction.py

```diff
@@ -113,8 +113,10 @@ def apply(self,
 
         for filename, this_diff in sorted(result.diffs.items()):
             to_filename = this_diff.rename if this_diff.rename else filename
+            to_filename = this_diff.create if this_diff.create else to_filename
             to_filename = '/dev/null' if this_diff.delete else to_filename
-            original_file = original_file_dict[filename]
+            original_file = ([] if this_diff.create
```

Comment:
above you use `this_diff.create is not False`. I think we can also agree on just testing against `bool(diff.create)`, as empty strings are invalid filenames anyway^^
Up to you if you wanna make this consistent or assuming an empty string for `diff.create == False` :+1: Or you can keep it as is ;)